polytunnel_navigation_actions/Obstacle[] obstacles
